content/05.discussion_and_conclusion.md

```diff
@@ -0,0 +1,15 @@
+## Discussion and Conclusions
+
+Filling out knowledge bases via manual curation can be an arduous and erroneous task [@doi:10.1093/bioinformatics/btm229].
+Using manual curation alone becomes impractical as the rate of publications continuously increases.
+Data programming is a paradigm that uses label functions to speed up the annotation process and can be used to solve this problem.
+However, creating useful label functions is an obstacle to this paradigm, which takes considerable time.
+We tested the feasibility of re-using label functions to reduce the number of label functions required for strong prediction performance.
+
+Our sampling experiment revealed that adding edge-specific label functions is better than adding off-edge label functions.
+An exception to this trend is using label functions designed from conceptually related edge types (using Gene-interacts-Gene (GiG) label functions to predict Compound-binds-Gene (CbG) sentences and vice versa).
+Furthermore, broad edge types such as Disease-associates-Gene (DaG) did not follow this trend as we found this edge to be agnostic to any tested label function source.
```

Comment:
What do you mean by agnostic? Didn't benefit from any label functions including its own?

Reply:
The opposite in this case. Performance increased regardless of the label function source. So agnostic in this case means that the edge type didn't prefer one source over the others.